src/ServiceV2/ResettingService.php

```diff
@@ -34,9 +34,9 @@ public function __construct(
     ) {
     }
 
-    public function handleEmailSend(User $user, ResetPasswordToken $token, string $locale): void
+    public function handleEmailSend(User $user, ResetPasswordToken $token): void
```

Comment:
Same here, `sendEmail` is simpler and clearer. "handle" as "process" is too generic and should be avoided if possible.